documentation/README.md

```diff
@@ -122,11 +122,20 @@ This option removes the selected block from your program
 ![Delete](./delete.png "Delete")
 
 # Help
-This option shows a help screen with information about the selected block. You can use this option to learn more about the block's functionality and how to use it in your projects.
+Help
+The "Help" option shows a detailed help screen with information about the selected block. It includes descriptions of the block's functionality, usage instructions, and examples to help users understand how to incorporate the block into their projects effectively.
 
 ![Help](./help.png "Help")
 
-By using the right-click context menu in Music Blocks, you can quickly perform common tasks and manipulate blocks on the workspace. This can help you to work more efficiently and effectively in your projects.
+Example Usage:
```

Comment:
I am not sure that the text beginning at Line 130 adds any value.

But you may want to mention that many help screens allow you do insert the example blocks into your program by clicking on the download icon (Load this block).